src/cls/IPM/Main.cls

```diff
@@ -2242,6 +2266,12 @@ ClassMethod Install(ByRef pCommandInfo) [ Internal ]
                 }
             }
             $$$ThrowOnError(##class(%IPM.Utils.Module).LoadQualifiedReference(tResult, .tParams, , tSynchronous))
+            if '$get(tParams("DeveloperMode")) {
+                set tModule = ##class(%IPM.Storage.Module).NameOpen(tModuleName)
```

Comment:
Seeding of update steps should be done in the activate phase per the design doc correct?